patches/minecraft/net/minecraft/loot/conditions/LootConditionManager.java.patch

```diff
@@ -0,0 +1,10 @@
+--- a/net/minecraft/loot/conditions/LootConditionManager.java
++++ b/net/minecraft/loot/conditions/LootConditionManager.java
+@@ -24,6 +24,7 @@
+    public static final LootConditionType field_237471_n_ = func_237475_a_("weather_check", new WeatherCheck.Serializer());
+    public static final LootConditionType field_237472_o_ = func_237475_a_("reference", new Reference.Serializer());
+    public static final LootConditionType field_237473_p_ = func_237475_a_("time_check", new TimeCheck.Serializer());
++   public static final LootConditionType LOOT_TABLE_ID = func_237475_a_("forge:loot_table_id", new net.minecraftforge.common.loot.LootTableIdCondition.Serializer()); // FORGE
```

Comment:
Not sure if this should be patched in? Probably better in ForgeMod or something no?

Reply:
That could be an alternative solution, but since we don't have a registry for `ILootCondition`s, when would be the best way to initialize it?
Mod constructor? In a new event for loot tables specifically?

Reply:
Oh right... well nvm then.

Reply:
I figured that vanilla registers them statically on bootup (-.-).
I reworked the addition into an AT and I register the class statically when Forge is initialized. A nice side effect of this is that other mods can now register their own conditions without having to AT or reflect themselves.
Of course, if an alternative ends up being available, like an event, the static init can be dropped in favor of the event.